docs/src/main/asciidoc/datasource.adoc

```diff
@@ -641,9 +614,10 @@ If the metrics collection for this datasource is disabled, all values result in
 
 To use tracing with a datasource, you need to add the xref:opentelemetry-tracing.adoc[`quarkus-opentelemetry`] extension to your project.
 
-You don't need to declare a different driver because you need tracing. If you use a JDBC driver, you need to follow the instructions in the OpenTelemetry extension xref:opentelemetry-tracing.adoc#jdbc[here].
+You don't need to declare a different driver because you need tracing.
```

Comment:
```suggestion
You do not need to declare a different driver because you need tracing.
```

Reply:
What does the "because you need tracing" part mean here, though? Is it like "Even if you need tracing, you don't need to declare a tracker". Or is it more along the lines of "Due to the fact you need tracing, you do not need to..."? Either way, the language should be clearer.

Reply:
```suggestion
You don't need to declare a different driver to enable tracing.
```